cirq-core/cirq/devices/superconducting_qubits_noise_properties_test.py

```diff
@@ -59,6 +59,10 @@ def default_props(system_qubits: List[cirq.Qid], qubit_pairs: List[Tuple[cirq.Qi
 
 
 class TestNoiseProperties(SuperconductingQubitsNoiseProperties):
+
+    # tell pytest there are no test methods here
+    __test__ = False
```

Comment:
What do you think about simply changing the name of the class instead?
Perhaps ExampleNoiseProperties?

Reply:
Sure, that will work as well.  Done.